src/pip/index.js

```diff
@@ -43,6 +43,17 @@ module.exports.create = function createPIPService(datapath, layers, localizedAdm
   // ie - _.intersection([1, 2, 3], [3, 1]) === [1, 3]
   layers = _.intersection(defaultLayers, _.isEmpty(layers) ? defaultLayers : layers);
 
+  // further refine the layers by filtering out layers for which there is not metafile
+  layers = layers.filter(layer => {
+    const filename = path.join(datapath, 'meta', `wof-${layer}-latest.csv`);
+
+    if (!fs.existsSync(filename)) {
+      logger.error(`unable to locate ${filename}`);
```

Comment:
To follow up on the conversation we had in person, I really believe this should have a configuration option to throw a fatal error (one that shuts down the entire admin lookup process). Otherwise a situation will inevitably arise where we run builds without all the admin data we wanted and don't find out until it's done, which wastes a lot of time.